requirements/base.txt

```diff
@@ -7,3 +7,4 @@ django-environ==0.10.0  # https://github.com/joke2k/django-environ
 django-allauth==0.57.0  # https://github.com/pennersr/django-allauth
 django-crispy-forms==2.0  # https://github.com/django-crispy-forms/django-crispy-forms
 crispy-bootstrap5==0.7  # https://github.com/django-crispy-forms/crispy-bootstrap5
+django-tables2==2.5.0  # https://github.com/jieter/django-tables2
```

Comment:
This needs to be in local.txt. But I just merged a PR that covers this, so you can just remove it.